_includes/concepts.md

```diff
@@ -130,7 +130,7 @@ This section ONLY deals with the spanning-tree topology, shown in the figure bel
 
 Communication on a one-to-one basis is usually only performed by clients, since most server-server traffic is not a result of servers talking only to each other.
 
-Servers MUST be able to send a message from any one client to any other. It is REQUIRED that all servers be able to send a message in exactly one direction along the spanning tree to reach any client. Thus the path of a message being delivered is the shortest path between any two points on the spanning tree.
+Servers are be able to send a message from any one client to any other. Outside of networking issues, all servers are always able to send a message in exactly one direction along the spanning tree to reach any client. Thus the path of a message being delivered is the shortest path between any two points on the spanning tree.
```

Comment:
```suggestion
Servers should be able to send a message from any one client to any other. Servers send a message in exactly one direction along the spanning tree to reach any client. Thus the path of a message being delivered is the shortest path between any two points on the spanning tree.
```
Just simplifies things and focuses on the conceptual description – no need to mention networking issues here because that won't affect the way messages are passed along the way conceptually.